config/migrations/20211119151700-move-reason-codes.sparql

```diff
@@ -0,0 +1,14 @@
+DELETE {
```

Comment:
These were inserted in the default graph by the migrations service so mu-cl-resources didn't return them.